Makefile

```diff
@@ -63,6 +63,13 @@ statusgo-ios-simulator: xgo	##@cross-compile Build status-go for iOS Simulator
 	$(GOPATH)/bin/xgo --image farazdagi/xgo-ios-simulator --go=$(GO) -out statusgo --dest=$(GOBIN) --targets=ios-9.3/framework -v $(shell build/testnet-flags.sh) ./lib
 	@echo "iOS framework cross compilation done."
 
+statusgo-library: ##@cross-compile Build status-go as static library for current platform
+	@echo "Building static library..."
+	@go build -buildmode=c-archive -o $(GOBIN)/libstatus ./lib
+	@ranlib $(GOBIN)/libstatus.a
```

Comment:
What's the purpose of `ranlib` here?

Reply:
@adambabik it's the normal workflow for creating static libraries — create C archive (`ar` tool, typically) and then add index of symbols in the archive (`ranlib`). So it's basically scan the `libstatus.a` and adds symbols index. Without it, linked (invoked by `node-gyp`, for example) will not be able to find symbols and link it.

Reply:
I haven't used that and all was ok. Are you sure Go compiled does not do that for you already?

Reply:
@adambabik hmm, I definitely added `ranlib` after investigating linking failure (it could not find a symbol). Apparently that was a case when I used `xgo` build instead of native `go build`. Maybe it does symbols stripping or something.

I'll remove it for now, then.

I also found that output target `libstatus` is incorrect and should be `libstatus.a` :) (another artifact from changing `xgo`-code into `go build` one).